app/controllers/api/v1/profiles_controller.rb

```diff
@@ -25,7 +25,33 @@ def auto_offline
   end
 
   def availability
-    @user.account_users.find_by!(account_id: availability_params[:account_id]).update!(availability: availability_params[:availability])
+    # Log the start of the method
+    Rails.logger.debug { 'Starting availability update' }
+
+    # Retrieve the account_id and availability from the parameters
+    account_id = availability_params[:account_id]
+    new_availability = availability_params[:availability]
+
+    # Log the retrieved parameters
+    Rails.logger.debug { "Parameters: account_id: #{account_id}, availability: #{new_availability}" }
+
+    # Find the AccountUser record
+    account_user = @user.account_users.find_by(account_id: account_id)
+
+    # Check if the AccountUser record is found
+    if account_user
+      # Log the current availability of the AccountUser
+      Rails.logger.debug { "Current availability of AccountUser (ID: #{account_user.id}): #{account_user.availability}" }
+
+      # Update the availability
+      update_result = account_user.update(availability: new_availability)
+
+      # Log the result of the update operation
+      Rails.logger.debug { "Update result: #{update_result}" }
+    else
+      # Log if the AccountUser record is not found
+      Rails.logger.debug { "No AccountUser found for user_id: #{@user.id} and account_id: #{account_id}" }
+    end
```

Comment:
This changes seems unrelated to the original issue. lets take it up on a separate PR if required.

Reply:
Hm, this might have been cause by the merge. Let me fix this.
Good catch!